entry_types/scrolled/package/.storybook/main.js

```diff
@@ -9,7 +9,23 @@ module.exports = {
       ...config,
       module: {
         ...config.module,
-        rules: addModuleOptionToCssLoader(config.module.rules),
+        rules: [
+          {
+            test: /\.svg$/,
+            use: [
+              {
+                loader: "babel-loader"
+              },
+              {
+                loader: "react-svg-loader",
+                options: {
+                  jsx: false
+                }
+              }
+            ]
+          },
+          ...addModuleOptionToCssLoader(removeFileLoader(config.module.rules))
+        ]
```

Comment:
A refined version of this has been merged as part of #1408 (https://github.com/codevise/pageflow/pull/1408/commits/2a1b5a0deb75a757c2d601decc53f2396e1e643c).